bids_conversion/convert_somato_data.py

```diff
@@ -144,30 +141,14 @@
 derivatives_dir = op.join(somato_path_bids, 'derivatives')
 subjects_dir_bids = op.join(derivatives_dir, 'freesurfer', 'subjects')
 
-# Run recon-all from freesurfer
-run_subprocess(['recon-all', '-i', t1w_nii, '-s', '01', '-all'])
+os.environ['SUBJECTS_DIR'] = subjects_dir_bids
```

Comment:
When we do this, we can also remove the copytree command below

Reply:
however, FreeSurfer usually places a symlink for `fsaverage` instead of copying over the directory ... that symlink would not work for all installations of FreeSurfer (different OS, different install location)

Reply:
but then again, I don't think we need to ship `fsaverage` with this data :-)

Reply:
I see, I don't think we need to ship `fsaverage` with this dataset. Note that I was getting an error the way you had it set up before, because my `SUBJECTS_DIR` was pointing to some directory that I did not have permission to. I could have changed it to point to the right directory but it seemed more work than just changing the code :)

Reply:
yes, definitely an improvement! ... no weird copying necessary :)